packages/daemon/src/app/AppsRepository.js

```diff
@@ -60,6 +67,39 @@ export type AppsRepositorySerialized = {
   ownApps: { [string]: OwnAppSerialized },
 }
 
+export const getContentsPath = (
+  env: Environment,
+  manifest: ManifestData,
+): string => {
+  return getAppContentsPath(env, manifest.id, manifest.version)
+}
+
+export const downloadAppContents = async (
+  app: App,
+  contentsPath: string,
+  bzz: BzzAPI,
```

Comment:
Could you make this the first argument please? Just for consistency with the other similar functions.